### PR TITLE
Cleaned shared ghost system

### PR DESCRIPTION
**Pull request by wrexbe**
## About the PR

1. Figured out what some things did, and commented them
2. Combined some event handlers
3. Renamed some params

### CODE REVIEW

**Comment by wrexbe** [comment]
Was messing with git, and killed the PR. woops.

File: Content.Shared/Ghost/SharedGhostSystem.cs [hunk 1]
```diff
@@ -9,70 +9,48 @@
...
-
-        public void SetCanReturnToBody(SharedGhostComponent component, bool canReturn)
-        {
-            if (component.CanReturnToBody == canReturn)
-            {
-                return;
-            }
-
-            component.CanReturnToBody = canReturn;
-            component.Dirty();
```
[the remark this answers is not in the document]
Comment (wrexbe) [reply Content.Shared/Ghost/SharedGhostSystem.cs:67]: The component does all the logic that used to be here already

[the remark this answers is not in the document]
Comment (metalgearsloth) [reply Content.Shared/Ghost/SharedGhostSystem.cs:67]: Ideally the system does it but that's a larger piece of work.

**Comment by metalgearsloth** [comment]
This does mean aghosts can attack people but I guess that's a feature.

**Pull request merged**